.gitpod.yml

```diff
@@ -0,0 +1,6 @@
+image:
+  file: .gitpod.Dockerfile
```

Comment:
This is needed only because of the previous custom image. According to gitpod's documentation (https://www.gitpod.io/docs/config-docker/), if we are using the gitpod's default docker then we don't need this.